junipervpn/tncc.py

```diff
@@ -79,7 +79,7 @@ def decode_0ce4(buf, indent):
 def decode_0ce5(buf, indent):
     s = struct.unpack(str(len(buf)) + "s", buf)[0]
     logging.debug('%scmd 0ce5 (string) %d bytes', indent, len(buf))
-    s = s.rstrip('\0')
```

Comment:
This all looks good to me, I'll go ahead and cherry-pick this commit onto `master` ASAP.